part-spec/passives.json

```diff
@@ -0,0 +1,237 @@
+                    "description": "nominal tolerance of a inductor",
+                    "comment": "units of %",
+                    "$ref": "definitions.json#/unit"
+                },
+                "ratedCurrent": {
+                    "description": "maximum continuous current the inductor can handle",
+                    "comment": "units of amps",
+                    "$ref": "definitions.json#/unit"
+                },
+                "saturationCurrent": {
+                    "description": "current where the inductor enters the magnetic state, and the inductance drops a specified amount",
+                    "comment": "units of amps",
+                    "$ref": "definitions.json#/unit"
+                },
+                "rmsCurrent": {
+                    "description": "DC current that produces an inductor temperature rise of 40 degrees Celsius",
+                    "comment": "units of amps",
+                    "$ref": "definitions.json#/unit"
+                },
+                "selfResonantFrequency": {
+                    "description": "frequency at which the inductor becomes a capacitor",
+                    "comment": "units of hertz",
+                    "$ref": "definitions.json#/unit"
+                },
+                "dcResistance": {
+                    "description": "DC resistance of the inductor",
+                    "comment": "units of ohms",
+                    "$ref": "definitions.json#/unit"
+                },
+                "temperatureCoefficient": {
+                    "description": "change in inductance when the temperature is changed",
+                    "comment": "units of henry/degree (Celsius or Kelvin)",
+                    "type": "number"
+                },
+                "minTemperature": {
+                    "description": "minimum temperature under which a inductor can be expected to reliably operate",
+                    "comment": "units of degrees (Celsius or Kelvin)",
+                    "$ref": "definitions.json#/unit"
+                },
+                "maxTemperature": {
+                    "description": "maximum temperature under which a inductor can be expected to reliably operate",
+                    "comment": "units of degrees (Celsius or Kelvin)",
+                    "$ref": "definitions.json#/unit"
+                },
+                "saturationCurve": {
+                    "description": "graph object to capture inductor saturation with current",
+                    "$ref": "definitions.json#/graphDefiniton"
+                },
+                "resonantFrequencyCurve": {
+                    "description": "graph object to capture inductor resonant frequency",
+                    "$ref": "definitions.json#/graphDefiniton"
+                },
+                "package": {
+                    "description": "package size of inductor",
+                    "$ref": "definitions.json#/package"
+                }
```

Comment:
complianceList property?

Reply:
the complianceList is part of componentID

Reply:
part of ComponentID